database/migration/index.js

```diff
@@ -0,0 +1,17 @@
+require("dotenv").config();
+const { sequelize, user, token } = require("../../models");
+
+(async () => {
+  try {
+    await sequelize.authenticate();
+    console.log("Connected to the database.");
```

Comment:
I guess this is fine for now, but subsequently we probably want to abstract this to some custom logger that will handle all our logging for now @Elozzy